chart/templates/configmap-env.yaml

```diff
@@ -15,6 +15,9 @@ data:
   DB_NAME: {{ .Values.postgresql.auth.database }}
   DB_POOL: {{ .Values.mastodon.sidekiq.concurrency | quote }}
   DB_USER: {{ .Values.postgresql.auth.username }}
+  {{- if .Values.postgresql.preparedStatements }}
+  PREPARED_STATEMENTS: {{ .Values.postgresql.preparedStatements | quote }}
```

Comment:
Not an expert on how to best organize a values file so thinking out loud here, but since the `postgresql` section is passed to the [bitnami chart](https://github.com/bitnami/charts/tree/main/bitnami/postgresql#parameters) and prepared statements is an option for Mastodon, shouldn't this technically go somewhere under `.Values.mastodon`?

Reply:
Good point, it should. Will update the PR